Client/Application/AppDelegate.swift

```diff
@@ -79,7 +79,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 
         log.debug("Getting profile…")
         let profile = getProfile(application)
-        appStateStore = AppStateStore()
+        appStateStore = AppStateStore(prefs: profile.prefs)
```

Comment:
can you file a followup bug whereby anything at all outside of profile that wants to access `profile.prefs` does so through the `AppStateStore`?